cmake_modules/BuildUtils.cmake

```diff
@@ -210,9 +184,12 @@ function(ADD_ICEBERG_LIB LIB_NAME)
             DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
   endif()
 
-  if(ARG_CMAKE_PACKAGE_NAME)
-    iceberg_install_cmake_package(${ARG_CMAKE_PACKAGE_NAME} ${LIB_NAME}_targets)
-  endif()
+  string(TOLOWER ${LIB_NAME} LIB_NAME_LOWER_CASE)
+  string(REPLACE "_" "-" LIB_NAME_DASH_SEPARATED_LOWER_CASE ${LIB_NAME_LOWER_CASE})
+  install(EXPORT ${LIB_NAME}_targets
```

Comment:
How about using the same export name (something like `iceberg_targets`) for all targets?
If we use the same export name, we can use `include("${CMAKE_CURRENT_LIST_DIR}/iceberg-targets.cmake")` in `iceberg-config.cmake`.